attestation-service/docs/policy.md

```diff
@@ -35,7 +35,12 @@ values provided by the RVPS.
 
 `input` are the TCB claims generated by the verifier.
 
-See the [default policy](../src/token/ear_default_policy.rego) for an example.
+See the [default policy](../src/token/ear_default_policy_cpu.rego) for an example.
+
+When using EAR tokens, the policy used to evaluate CPU evidence should end in `_cpu`.
```

Comment:
A naive idea is to have only one default policy. It could have different parts to filter all supported devices. This would help to avoid the complex policy set-ups for users.